data-model/pom.xml

```diff
@@ -224,10 +216,6 @@
                 </exclusion>
             </exclusions>
         </dependency>
-        <dependency>
-            <groupId>org.bouncycastle</groupId>
-            <artifactId>bcmail-jdk15to18</artifactId>
-        </dependency>
         <dependency>
```

Comment:
Removed because we don't exclude `bcmail` from `tika-parsers-standard-package` anymore so we don't need to re-declare this anymore IMO.